index.css

```diff
@@ -0,0 +1,15 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.component-container {
+  margin: 40px 20px;
+}
+
+.component-heading {
+  margin-bottom: 20px;
+}
```

Comment:
use rem where ever possible. if you are using font-size: 62.5% then use px only when size is less than 10px else use rem where ever possible.

Reply:
@Creator-singh pls review again